PWGEM/PhotonMeson/Core/V0PhotonCut.h

```diff
@@ -138,26 +138,22 @@ class V0PhotonCut : public TNamed
       if (mIsWithinBP && !IsSelectedTrack(track, V0PhotonCuts::kIsWithinBeamPipe)) {
         return false;
       }
-      if (!track.hasITS() && !track.hasTPC()) { // track has to be ITSonly or TPConly
+      if (!track.hasITS() && !track.hasTPC()) { // track has to be ITSonly or TPConly or ITS-TPC
         return false;
       }
 
-      bool isITSTPC = track.hasITS() & track.hasTPC();
       bool isITSonly = track.hasITS() & (!track.hasTPC() & !track.hasTOF() & !track.hasTRD());
-      bool isTPConly = track.hasTPC() & (!track.hasITS() & !track.hasTOF() & !track.hasTRD());
 
-      if (isITSTPC) {
-        if (!CheckITSCuts(track) || !CheckTPCCuts(track)) {
-          return false;
-        }
-      } else if (isITSonly) {
+      if (isITSonly) {
         if (!CheckITSCuts(track)) {
           return false;
         }
-      } else if (isTPConly) {
+      } else if (track.hasTPC()) {
         if (!CheckTPCCuts(track)) {
           return false;
         }
+      } else { // remove ITS-TRD, ITS-TOF, ITS-TRD-TOF that are unrealistic tracks.
+        return false;
```

Comment:
The check down below 
```c++
else if (track.hasTPC()) {
         if (!CheckTPCCuts(track)) {
           return false;
         }
```
now can also include tracks with TRD and/or TOF, right?

Reply:
TRD and TOF are not required at all. But, they are not "rejected".

Reply:
Yeah but previously you had an option for actual TPC only, so where TRD and TOF were rejected. Is the removal of this what we actually want?

Reply:
TRDonly and TOFonly tracks are always rejected. In my previous commit, I noticed cuts were not applied to TPC-TRD-TOF, TPC-TOF, TPC-TRD. It is correct now.